5-infrastructure/modules/base-env/boa-gke.tf

```diff
@@ -122,4 +124,38 @@ module "clusters" {
       node_metadata      = "GKE_METADATA_SERVER"
     }
   ]
+  node_pools_oauth_scopes = {
+    "all" : [
+      "https://www.googleapis.com/auth/cloud-platform",
+      "https://www.googleapis.com/auth/logging.write",
+      "https://www.googleapis.com/auth/monitoring",
+      "https://www.googleapis.com/auth/devstorage.read_only"
+    ],
+    "default-node-pool" : []
+  }
+}
+
+resource "google_binary_authorization_policy" "policy" {
+  count                         = var.enable_bin_auth_policy ? 1 : 0
+  global_policy_evaluation_mode = "ENABLE"
+  default_admission_rule {
+    evaluation_mode         = "REQUIRE_ATTESTATION"
+    enforcement_mode        = "ENFORCED_BLOCK_AND_AUDIT_LOG"
```

Comment:
lets switch `enable_bin_auth_policy` to `enforce_bin_auth_policy` and switch between `ENFORCED_BLOCK_AND_AUDIT_LOG` and `DRYRUN_AUDIT_LOG_ONLY `